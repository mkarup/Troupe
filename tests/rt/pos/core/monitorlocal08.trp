let val p = spawn (fn () => receive [ hn x => "crash" ^ 1 ])
    val monRef = monitorlocal (p, `{}`)
    val _ = send (p, 42)
in
  receive [ hn (str, monRef', from, reason) =>
            case reason of
                (1, errStr) => true
              | _ => false
          ]
end