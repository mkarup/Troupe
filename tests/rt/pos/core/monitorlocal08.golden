2024-12-11T12:25:44.680Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread 102498a7-2ced-40c3-bfc3-9805294c8cc8@{}%{}
>> value 1 is not a string
>>> Main thread finished with value: true@{}%{}
